dashboard/app/views/projects/index.html.haml

```diff
@@ -8,6 +8,12 @@
   // TODO Erin B - remove when we have enough featured Applab and Gamelab projects and/or profanity filter in place.
   - projects_data[:projectValidator] = current_user.permission?  UserPermission::PROJECT_VALIDATOR
   - projects_data[:canShare] = !current_user.sharing_disabled?
+  // We're going to run an A/B experiment to compare (un)publishing from the
+  // quick actions dropdown and from a button in the published column.
+  // As part of the experiment, we want to log users id.
+  // TODO (Erin B.) delete this data when we
+  // determine the experiment outcome.
+  - projects_data[:usedId] = current_user.id
```

Comment:
Doesn't look like you need this anymore.

Reply:
Oh I see, you're using it in https://github.com/code-dot-org/code-dot-org/pull/24322/files. I think you meant `userId`, not `usedId` though.

Reply:
yes, good catch!